Automattic/jetpack
# Moving utility and ajax to videopress package

#### Changes proposed in this Pull Request:

* Moves utlitiy functions and ajax callbacks to the VideoPress package

#### Other information:


#### Jetpack product discussion


NA

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Activate Jetpack plugin
* Add a new post
* Make sure the current videopress block (regular video block) and the new VideoPress Beta block are still both working
* Test uploading a video in both blocks


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202561463041781
